### ISSUE TITLE
Song change confirmations

### ISSUE THREAD

**Issue by danirabbit** [elementary/notifications#45]
Instead of doing song change confirmations at the app level, we should do it here in the server so that they don't get sent to the notification center, apps don't have to implement them, and we can incorporate MPRIS controls

**Comment by lenemter** [comment]
I'm -1 on this. TBH I've never saw song change confirmations but I suppose they're annoying af

**Comment by danirabbit** [comment]
Yeah I think this was a more common behavior a long time ago. Media controls are ubiquitous enough now that this isn't popular anymore I think
